branchpro/apps/_dash_app.py

```diff
@@ -53,6 +53,134 @@
 </html>
 """
 
+df = pd.DataFrame({
```

Comment:
Let's add brief comments before df, inc_modal, rows, and si_modal, so anyone reading this file looking for the base class knows that we are just writing data here.